packages/client/src/v2-events/features/workqueues/EventOverview/components/EventHistory/EventHistoryDialog/components/PrintCertificate.tsx

```diff
@@ -66,7 +69,7 @@ export function PrintCertificate({
   // When we merge phase-3 branch here, we could try refactoring this to be a shared frontend module/function.
   const content = formPages.flatMap((page) => {
     const fields = page.fields
-      .filter((f) => isFieldVisible(f, annotation))
+      .filter((f) => isFieldVisible(f, annotation, userContext))
```

Comment:
Can you not use the `isFieldVisible` from the `useValidationFunctionsWithContext()` here? I think this same applies to all usage of these validations functions under `packages/client`